tests/test_cloud_for_customers.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for CloudForCustomers source class"""
+
+from viadot.utils import get_credentials
+from viadot.sources.cloud_for_customers import CloudForCustomers
+import pandas as pd
+
+
+url = "https://my336539.crm.ondemand.com/sap/c4c/odata/v1/c4codataapi/"
```

Comment:
This should not be provided here but put in `conftest` similar as other sensitive settings.

Reply:
Done